src/cider/nrepl/middleware/apropos.clj

```diff
@@ -2,12 +2,36 @@
   "Search symbols and docs matching a regular expression"
   {:author "Jeff Valk"}
   (:require [cider.nrepl.middleware.util.error-handling :refer [with-safe-transport]]
-            [orchard.apropos :as apropos]))
+            [orchard.apropos :as apropos]
+            [cider.nrepl.middleware.util.nrepl :as util.nrepl]))
 
 ;;; ## Middleware
 
 (defn apropos [msg]
-  {:apropos-matches (apropos/find-symbols msg)})
+  {:apropos-matches
+   (apropos/find-symbols
+    (cond-> msg
+
+      (:privates? msg)
```

Comment:
You might want to add some notes about compatibility code that will be removed down the road. Otherwise we'll likely forget to do this. :-)